setup.py

```diff
@@ -22,7 +22,7 @@
 }
 statuses = [ '1 - Planning', '2 - Pre-Alpha', '3 - Alpha',
     '4 - Beta', '5 - Production/Stable', '6 - Mature', '7 - Inactive' ]
-py_versions = '3.6 3.7 3.8 3.9 3.10'.split()
+py_versions = '3.7 3.8 3.9 3.10 3.11'.split()
```

Comment:
Let's remove '3.7' since `settings.ini` declares `min_python = 3.8`.

Reply:
We can have extra entries since they're filtered below using the min_python from the settings but I just removed it in 5434db9